specutils/tests/conftest.py

```diff
@@ -4,30 +4,33 @@
 
 import tempfile
 
-
-REMOTE_DATA_IDS = {
-    'cos_fuv': ('1481119', 'COS_FUV.fits')
-}
-
-REMOTE_DATA_NAMES = {
-    'COS_FUV.fits': '1481119'
-}
-
 ACCESS_TOKEN = os.environ.get("ACCESS_TOKEN")
 
 
-@pytest.fixture(scope='module', params=REMOTE_DATA_IDS.values(), ids=REMOTE_DATA_IDS.keys())
-def remote_data(request):
+@pytest.fixture(scope='module')
+def remote_data_path(request):
```

Comment:
Maybe not for this PR, but if there's any chance we'd want to use this in `specviz` as well maybe this should be factored into a pytest plugin so it can be reused.